### DISCUSSION TITLE
Files husky/prepare-commit-msg and lefthook.yml created in root of App

### DISCUSSION THREAD

**Discussion by wilsolutions** [callstack/react-native-builder-bob#560] [Q&A]
Hi,

I have created a native module as usual, however I've just noticed that these files .husky/prepare-commit-msg and lefthook.yml are created in the App once I add the new created native module.

Please, anyone knows why and how to prevent?

I feel like it is super intrusive, I don't these files to be created in the App after I simply add a native module.

Thanks

**Comment by wilsolutions** [comment]
For reference in case anyone needs, it turns out I just needed to remove it from the package.json:

`"@evilmartians/lefthook": "^1.5.0",`

**Comment by satya164** [comment]
Sounds like you installed the library using a relative path. It's not a recommended way to install packages.

If you just want to test it, then you can test in the example app. If you want to test in another app, then you can follow these options https://callstack.github.io/react-native-builder-bob/faq#how-to-test-the-library-in-an-app-locally

If you want to use the library in a production app then you should publish it to a registry such as npm or GitHub package registry. If you don't want to publish it, then you can use a local library https://callstack.github.io/react-native-builder-bob/create#local-library
